lib/src/telegram/models/chat_permissions.dart

```diff
@@ -41,6 +41,5 @@ class ChatPermissions {
   });
   factory ChatPermissions.fromJson(Map<String, dynamic> json) =>
       _$ChatPermissionsFromJson(json);
-  Map<String, dynamic> toJson(Map<String, dynamic> json) =>
-      _$ChatPermissionsToJson(this);
+  Map<String, dynamic> toJson() => _$ChatPermissionsToJson(this);
```

Comment:
please revert this change, it will be fix in #125

Reply:
Done